fdbserver/fdbserver.actor.cpp

```
@@ -1553,7 +1553,11 @@ int main(int argc, char* argv[]) {
 		} else {
 #ifndef TLS_DISABLED
 			if ( tlsVerifyPeers.size() ) {
-			  tlsPolicy->set_verify_peers( tlsVerifyPeers );
+				if (!tlsPolicy->set_verify_peers( tlsVerifyPeers )) {
```

Throw instead of bool return.